cmd/lakectl/cmd/root.go

```diff
@@ -94,6 +94,12 @@ type Configuration struct {
 		// setting FixSparkPlaceholder to true will change spark placeholder with the actual location. for more information see https://github.com/treeverse/lakeFS/issues/2213
 		FixSparkPlaceholder bool `mapstructure:"fix_spark_placeholder"`
 	}
+	// Experimental - Use caution when enabling experimental features. It should only be used after consulting with the lakeFS team!
+	Experimental struct {
+		Local struct {
+			UnixPerm bool `mapstructure:"unix_permissions"`
```

Comment:
The name sounds too general to me.
Maybe `EnforceUnixPerm`, or `SyncUnixPerm`?

Reply:
Sounds good